walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -25,35 +25,51 @@
     if (request.getDuration() > TimeRange.WHOLE_DAY.duration()) {
       return availableSlots;
     }
+    List<TimeRange> availableSlotsOptional = new ArrayList<>();
 
     availableSlots.add(TimeRange.WHOLE_DAY);
+    availableSlotsOptional.add(TimeRange.WHOLE_DAY);
     for (Event event : events) {
       TimeRange eventTimeRange = event.getWhen();
-      Collection<String> attendees = request.getAttendees();
```

Comment:
Keep this line (and similar for optional attendeed), and this should be moved before the loop (I missed this previously) - since request is the same for every event.

Reply:
+ request.getDuration() as well